### DISCUSSION TITLE
Proactive message to user using composer

### DISCUSSION THREAD

**Discussion by jrussomano** [microsoft/BotFramework-Composer#7650] [Q&A]
Hi, is there a way to send a proactive message to a user (I have the conversation id) from a backend appliation? I know it is possible using bot framework sdk, but I could not find a way using composer.

Thanks.

**Comment by AntonioChiancone** [comment]
From composer you can use Events. From FE you can send an event to chatbot or manage the Activity Events (e.g. Typing Activity if Now == 09:00:00 then Send Response "Goodmorning"
